src/master/grafana/dashboards/NVIDIA GPU/gpu-device-counters.json

```diff
@@ -1774,8 +1777,8 @@
       {
         "current": {
           "selected": false,
-          "text": "moneoVM",
-          "value": "moneoVM"
+          "text": "ub22h3e8f000003",
```

Comment:
Is this a default VM name?

Reply:
good catch. it doesn't break anything but it looks nicer